src/apps/TicTacToe.js

```diff
@@ -0,0 +1,148 @@
+import { useState } from "react";
+import styled from "styled-components";
+import { InstantApp } from "./types";
+
+//------------Styled Components------------
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+const RestartButton = styled.button`
+  background: transparent;
+  cursor: pointer;
+
+  padding: 0.5rem 1.5rem;
+  border-radius: 7px;
+`;
+
+const BoardContainer = styled.div`
+  margin: 0.5rem;
+`;
+
+//=========================================
+
+function Square(props) {
+  const StyledSquare = styled.button`
+    background: #fff;
+    border: 1px solid #999;
+    float: left;
+    font-size: 24px;
+    font-weight: bold;
+    line-height: 34px;
+    height: 34px;
+    margin-right: -1px;
+    margin-top: -1px;
+    padding: 0;
+    text-align: center;
+    width: 34px;
+  `;
+
+  return <StyledSquare onClick={props.onClick}>{props.value}</StyledSquare>;
+}
+
+const Board = () => {
+  const [squares, setSquares] = useState(Array(9).fill(null));
```

Comment:
TIL `Array(N).fill()` 🤯 

Was looking for the Pythonic syntax in JS the other day - `[0]*9`